src/Configuration.ts

```diff
@@ -52,13 +59,27 @@ const configurations: { [name: string] : Configuration } = {
     deployFromEnvironment: Statics.gnBuildEnvironment,
     deployToEnvironment: Statics.appDevEnvironment,
     includePipelineValidationChecks: false,
-    allowedAccountIdsToPublishToSNS: [Statics.acceptanceWebformulierenAccountId],
+    allowedAccountIdsToPublishToSNS: [
+      Statics.acceptanceWebformulierenAccountIdOldLz,
+      Statics.acceptanceWebformulierenAccountId,
+    ],
+    subscribeToTopicArns: [
+      'arn:aws:sns:eu-west-1:315037222840:eform-submissions',
```

Comment:
waarom staan hier hardcoded account id's?

Reply:
Ik heb hier de SNS topic ARNs toegevoegd omdat ik een lijst van ARNs wil opgeven. Door dit via SSM te doen moet je aan de haal met CloudFormation functions om de lijst op te splitsen (`Fn.split()` etc.). De plek waar deze ARNs gebruikt worden geeft vervolgens een foutmelding omdat deze een CloudFormation parameter referentie verwacht of een ARN string (niet een CloudFormation token).